src/urllib3/connectionpool.py

```diff
@@ -398,7 +397,7 @@ def _make_request(
         # Receive the response from the server
         try:
             httplib_response = conn.getresponse()
-        except (SocketTimeout, BaseSSLError, SocketError) as e:
+        except (BaseSSLError, OSError) as e:
```

Comment:
`SocketTimeout` (i.e. `socket.timeout) is a subclass of `OSError` so including it would be redundant.
https://docs.python.org/3/library/socket.html#socket.timeout